### ISSUE TITLE
Brewing Stands can consume Slimefun Items

### ISSUE THREAD

**Issue by andris155** [TheBusyBiscuit/Slimefun4#563]
## Description
Describe your Issue AS DETAILED AS POSSIBLE.
Magical Essence works on Brewing.

## Server Log
Upload your FULL SERVER LOG (/logs/latest.log) to http://pastebin.com/ and paste your link in here.
Without this File, we may not be able to help you at all.
I send if need.

## Evidence / Proof
Attach as many Screenshots or Videos if possible.
Anything that helps us understand the Issue better, is gonna get your Issue fixed even faster.
![34485928-5f11430a-efcd-11e7-9382-cf970e0fead3](https://user-images.githubusercontent.com/8324198/34566236-2747cd58-f15e-11e7-88d0-0a0188c17402.png)

![34485931-665a5d4a-efcd-11e7-90cf-fac3088af0e4](https://user-images.githubusercontent.com/8324198/34566337-71e46cfe-f15e-11e7-8268-a1dedb7c7563.png)

## Environment
We need the EXACT VERSIONS of the Software you are using.
This includes your Minecraft Version, your CS-CoreLib version and your Slimefun Version.
If you cannot figure out what versions you are running, then please run /sf versions and
show us the output of that command.
```
04.01 14:48:26 [Server] INFO Paper v1_12_R1
04.01 14:48:26 [Server] INFO CS-CoreLib v1.5.16
04.01 14:48:26 [Server] INFO Slimefun v4.1.12
04.01 14:48:26 [Server] INFO Installed Addons (4)
04.01 14:48:26 [Server] INFO ExtraGear v1.1.1
04.01 14:48:26 [Server] INFO SlimefunLuckyBlocks v1.1
04.01 14:48:26 [Server] INFO ExoticGarden v1.6.3
04.01 14:48:26 [Server] INFO ElectricSpawners v1.0
```

**Comment by Poslovitch** [comment]
Comes from Exotic Garden. Shouldn't be reported here

**Comment by Poslovitch** [comment]
Hmm, well. We should provide protection inside Slimefun, though

**Comment by TheBusyBiscuit** [comment]
1. I didn't know brewing now requires blaze powder. Shows how long I haven't played Minecraft.
2. Yes, @Poslovitch just like slimefun prevents items from being put into crafting tables. We should prevent Items registered in Slimefun from being used in Brewing Stands.

**Comment by VoidAngel** [comment]
Fixed in #674
